### ISSUE TITLE
wpm --load source code

### ISSUE THREAD

**Issue by anurag3301** [cslarsen/wpm#64]
I just installed wpm, It works totally fine. I wanted to do typing practice on source code, the source code was opening fine and I was able to type. But I am facing one issue, the application is not registering a new line '\n'.

![gif](https://s6.gifyu.com/images/2021-06-16-20-15-45-1.gif)

**Comment by scopalaffairs** [comment]
https://github.com/cslarsen/wpm/issues/49 addresses the same issue.
There is a PR for that, which, at the time of writing, is open since over a year now.
I don't think this repository is under active development currently.

I picked the commit https://github.com/cslarsen/wpm/pull/56 applying it locally.
It's not as comfortable as sourcing it directly from the registry, but it works for my purposes, which, supposedly are similar to yours.

Apparently, the line breaks are a little bit messed up after a typing session in the terminal, but the benefit of loading custom quotes outweighs that for me :)

**Comment by anurag3301** [comment]
Yeah tried that PR seems to work but has some issue. Whatever the project is dead.
Can you recommend any other project like this where I can practice writing source code by importing them?
